addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -435,6 +435,13 @@ export class WebsitePreview extends Component {
         // exist, so we do not replace the iframefallback content.
         // The iframefallback is hidden in test mode
         if (!this.websiteContext.edition && this.iframe.el.contentDocument.body && this.iframefallback.el) {
+            // Remove autoplay in all youtube videos urls so videos are not playing in the background
+            const players = this.iframe.el.contentDocument.querySelectorAll('[id^=youtube-player]');
```

Comment:
`youtube-player` is something related to slides only, let's patch it in slides, maybe introducing an util hook here to manipulate the iframe fallback?
So anyone can do whatever they need with it, I guess it's a matter of time before such other issues are found and we won't want them all to be added here in a non modular way.

Also, shouldn't we do that too with the media_iframe_video > urls containing autoplay for regular pages?
I guess the sound is not enabled by default but the video is still loading and playing in the background for no good reason, isn't it?

Reply:
Hello @rdeodoo :slightly_smiling_face:  Thanks for the review :+1: 
I updated the fix regarding your suggestions, is it what you meant? (not sure though how to implement the util hook with `Fullscreen` widget in website_slides...)

In website slide, I still targeted only the YouTube iframes since did not find any other service that we set the autoplay on (tested with vimeo, dailymotion, youtube, drive)

For `media_iframe_video`, I still let the fix in `website` since mostly used in `web_editor` that inherit from `website` (but also used in `website` module)